rules-engine/src/rules_engine/engine.py

```diff
@@ -97,28 +97,100 @@ def __init__(
         heat_sys_efficiency: float,
         initial_balance_point: float = 60,
         thermostat_set_point: float = 68,
+        has_boiler_for_dhw: bool = False,
+        same_fuel_dhw_heating: bool = False,
     ):
         self.fuel_type = fuel_type
         self.heat_sys_efficiency = heat_sys_efficiency
         self.balance_point = initial_balance_point
         self.thermostat_set_point = thermostat_set_point
+        self.has_boiler_for_dhw = has_boiler_for_dhw
+        self.same_fuel_dhw_heating = same_fuel_dhw_heating
 
     def initialize_billing_periods(
-        self,
-        temps: List[List[float]],
-        usages: List[float],
-        avg_non_heating_usage: float = 0,
+        self, temps: List[List[float]], usages: List[float], inclusion_codes: List[int]
     ) -> None:
         """Eventually, this method should categorize the billing periods by
         season and calculate avg_non_heating_usage based on that. For now, we
         just pass in winter-only heating periods and manually define non-heating
         """
         # assume for now that temps and usages have the same number of elements
-        self.bills = []
-        self.avg_non_heating_usage = avg_non_heating_usage
 
+        self.bills_winter = []
+        self.bills_summer = []
+        self.bills_shoulder = []
+
+        # winter months 1; summer months -1; shoulder months 0
```

Comment:
I know this maps directly to the sheet but I wonder if for clarity it would be helpful to represent this as an enum?  We could even use the same inclusion code values, but name them e.g.:

```py
class Season(Enum):
  WINTER = 1
  SUMMER = -1
  SHOULDER = 0
```